src/main/java/org/llorllale/cactoos/matchers/InputHasContent.java

```diff
@@ -29,27 +29,18 @@
 import org.cactoos.Input;
 import org.cactoos.Text;
 import org.cactoos.text.TextOf;
-import org.cactoos.text.UncheckedText;
-import org.hamcrest.Description;
 import org.hamcrest.Matcher;
-import org.hamcrest.TypeSafeMatcher;
 
 /**
  * Matcher for the input.
- *
  * @since 0.11
- * @todo #7:30min All the matchers should implement describeMismatchSafely and
- *  their tests must verify that the implementation of the descriptions
- *  are satisfactory. The matchers should not expose publicly the xxxSafely
- *  method and the tests should rely on actual real use with assertThat.
- *  See ScalarHasValueTest for an example of a satisfactory result.
+ * @checkstyle IndentationCheck (500 lines)
```

Comment:
@Iprogrammerr remove `@checkstyle IntentationCheck (500 lines)` and deactivate the indentation check on the specific lines that are non-compliant. Use the `// @checkstyle IndentationCheck (1 line)` syntax.